api/envoy/config/metrics/v3/stats.proto

```diff
@@ -406,3 +406,30 @@ message HystrixSink {
   // <https://github.com/Netflix/Hystrix/wiki/Metrics-and-Monitoring#hystrixrollingnumber>`_.
   int64 num_buckets = 1;
 }
+
+// Stats configuration proto schema for *envoy.stat_sinks.graphite_statsd* sink.
```

Comment:
could you make this ` ``envoy.stat_sinks.graphite_statsd`` ` - using `*` is wrong it should be a literal

Reply:
done